_topics/en/adaptor-signatures.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Adaptor signatures
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - Signature adaptors
+  - Scriptless scripts
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Contract Protocols
+  - Privacy Enhancements
+  - Scripts and Addresses
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Adaptor signatures** (also called **signature adaptors**) are
+  auxiliary signature data that commit to a hidden value.  When an
+  adaptor is combined with a corresponding signature, it reveals
+  the hidden value.  Alternatively, when combined with the hidden value,
+  the adaptor reveals the signature.  Other people may create secondary
+  adaptors that reuse the commitment even if they don't know the hidden
+  value. This makes adaptors a powerful tool for implementing locking in
+  bitcoin contracts.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  Contracts in Bitcoin often require a locking mechanism to ensure the
+  atomicity of a set of payments---either all the payments succeed or
+  all of them fail.  This locking has traditionally been done by having
+  all payments in the set commit to the same hash digest preimage; when
+  the party who knows the preimage reveals it onchain, everyone else
+  learns it and can unlock their own payments.  Commonly used *hashlocks*
+  in Bitcoin consume about 67 bytes <!-- push32:1, preimage:32,
+  push32:1, hash:32, OP_CHECKEQUALVERIFY:1 --> and reveal the link
+  between the set of payments because they all use the same preimage and
+  digest.
+
+  By comparison, signature adaptors never need to be published onchain.   To anyone
+  without a corresponding adaptor, a signature created with an adaptor looks
+  like any other digital signature, giving adaptors significant efficiency
+  and privacy advantages over hashlocks.
+
+  ### Example
+
+  The multiple uses of signature adaptors can be seen in a simple
+  coinswap protocol.  For example, Alice can give Bob an adaptor
+  for an unsigned transaction that promises to pay him 1 BTC.  An
+  adaptor by itself can't be used as a BIP340 signature, so
+  Alice hasn't paid Bob yet.
+
+  What the adaptor does provide Bob is a commitment to Alice's hidden
+  value.  This commitment includes a parameter Bob can use to create a
+  second adaptor that commits to the same hidden value as Alice's
+  adaptor.  Bob can make that commitment even without knowing Alice's
+  hidden value or his own signature for that commitment.  Bob gives
+  Alice his adaptor and a corresponding unsigned transaction that
+  promises to pay her 1 BTC.
```

Comment:
The interesting thing is that from this adaptor "commitment" is that if you have the "hidden value" you can extract something valuable from it. As far as my reading of cryptography goes  whenever we have have such a commitment it's called an "encryption" and the "hidden value" is called a "decryption key". This would make the "parameter" that you can re-use to create another "commitment" a "public encryption key".

I understand that this comment is probably very helpful at this stage but I couldn't resist pointing this out as I think this paragraph demonstrates how difficult it is to talk about this without using public key encryption terminology.